core/src/test/java/io/snyk/plugins/artifactory/scanner/ScannerModuleTest.java

```diff
@@ -324,7 +324,7 @@ void testScanPythonItem_noVulns() throws Exception {
 
     TestResult tr = testResultCaptor.getValue();
     assertTrue(tr.success);
-    assertEquals(0, tr.dependencyCount);
+    assertEquals(1, tr.dependencyCount);
```

Comment:
This was a bug on our end, that is now fixed with the `topLevelDeps` solution. It should be `1`, there's not 0 dependencies.